manage/deploying/production/ubuntu_production.rst

```diff
@@ -49,27 +49,11 @@ Then, install the platform's build kit, nginx, and supervisor:
 Step 2: Install Plone
 ---------------------
 
-Check `http://plone.org/products/plone <http://plone.org/products/plone>`_. Follow the `Download` link to get to the latest release. Copy the URL for the `Unified Installer`. Substitute that URL below:
-
-.. code-block:: console
-
-    wget --no-check-certificate https://launchpad.net/plone/5.0/5.0/+download/Plone-5.0-UnifiedInstaller.tgz
-
-Unpack, change into the unpack directory and run the installer:
-
-.. code-block:: console
-
-    tar -xf Plone-5.0-UnifiedInstaller.tgz
-    cd Plone-5.0-UnifiedInstaller
-    sudo ./install.sh
-
-This will install Plone to /usr/local/Plone. There are installer options to put it elsewhere, and the install script will guide you through them. Use ``install.sh --help`` to see all command-line switches.
-
-
+Please take some time and read the cheapter about :doc:`installation </manage/installing/installing/index>`.
 
 .. note::
 
-    Note that this is `root` installation. The installer will create special system users to build and run Plone.
+    Note that this is `root` installation `sudo ./install.sh`. The installer will create special system users to build and run Plone.
```

Comment:
"create special system users"? how many? is it just one, i.e. the sentence should be "create a special system user"?